docs/tracing.md

```diff
@@ -0,0 +1,208 @@
+# Tracing on Amazon EKS
+
+[Distributed tracing](https://aws-observability.github.io/observability-best-practices/signals/traces/)
+helps you have end-to-end visibility between transactions in distributed nodes.
+The `eks-monitoring` module is configured  by default to collect traces into
+[AWS X-Ray](https://docs.aws.amazon.com/xray/latest/devguide/aws-xray.html).
+
+The AWS Distro for OpenTelemetry collector is configured to receive traces
+in the OTLP format (OTLP receiver), using the OpenTelemetry SDK or
+auto-instrumentation agents.
+
+!!! note
```

Comment:
This is not accurate. We should say to disable remove `XrayAddOn` from AWS native or `XrayAdotAddOn` from Mixed and Open Source Observability Accelerators and reference CDK accelerator `lib` folder URL.